### PR TITLE
[DO NOT MERGE] Workflow API docs draft

### PR DESCRIPTION
**Pull request by hhunter-ms**
Thank you for helping make the Dapr documentation better!

**Please follow this checklist before submitting:**
- [ ] Commits are signed with Developer Certificate of Origin (DCO - [learn more](https://docs.dapr.io/contributing/contributing-overview/#developer-certificate-of-origin-signing-your-work))
- [ ] [Read the contribution guide](https://docs.dapr.io/contributing/docs-contrib/contributing-docs/)
- [ ] Commands include options for Linux, MacOS, and Windows within codetabs
- [ ] New file and folder names are globally unique
- [ ] Page references use shortcodes instead of markdown or URL links
- [ ] Images use HTML style and have alternative text
- [ ] Places where multiple code/command options are given have codetabs

In addition, please fill out the following to help reviewers understand this pull request:

### CODE REVIEW

File: daprdocs/content/en/concepts/faq.md [hunk 1]
```diff
@@ -27,6 +27,16 @@ Creating a new actor follows a local call like `http://localhost:3500/v1.0/actor
 
 The Dapr runtime SDKs have language-specific actor frameworks. For example, the .NET SDK has C# actors. The goal is for all the Dapr language SDKs to have an actor framework. Currently .NET, Java, Go and Python SDK have actor frameworks.
 
+## Workflows
+
+### Differences between Dapr Workflows, authoring APIs, and the management API
```
Comment (msfussell) [inline daprdocs/content/en/concepts/faq.md:32]: This is not a useful FAQ question IMO.

File: daprdocs/content/en/concepts/faq.md [hunk 2]
```diff
@@ -27,6 +27,16 @@ Creating a new actor follows a local call like `http://localhost:3500/v1.0/actor
...
+
+TODO
+
+### What is the relationship between Dapr Workflows and the Durable task framework?
```
Comment (msfussell) [inline daprdocs/content/en/concepts/faq.md:36]: Is this useful? What is this attempting to explain?

File: daprdocs/content/en/reference/api/workflow_api.md [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+| `spec/metadata` | Additional metadata parameters specified by workflow component |
+
+
+
+## Supported workflow methods
+
+### POST start workflow request
+```bash
+POST http://localhost:3500/v1.0-alpha1/workflows/{workflowComponent}/{workflowType}/{instanceId}/start
+```
```
Comment (msfussell) [inline daprdocs/content/en/reference/api/workflow_api.md:35]: There needs to be examples here for Dapr and Temporal

If I use Dapr workflow with the HTTP API, I presume that I will have to specify the component type. Given this is in Dapr how do I use this API?
 But what is the component?
POST http://localhost:3500/v1.0-alpha1/workflows/{workflowComponent}/workflow.dapr/123/start

File: daprdocs/content/en/reference/api/workflow_api.md [hunk 4]
```diff
@@ -0,0 +1,120 @@
...
+| `metadata.name` | The name of the workflow component. |
+| `spec/metadata` | Additional metadata parameters specified by workflow component |
+
+
+
+## Supported workflow methods
+
+### POST start workflow request
+```bash
+POST http://localhost:3500/v1.0-alpha1/workflows/{workflowComponent}/{workflowType}/{instanceId}/start
```
Comment (msfussell) [inline daprdocs/content/en/reference/api/workflow_api.md:34]:
```suggestion
POST http://localhost:3500/v1.0-alpha1/workflows/{name}/{workflowType}/{instanceId}/start
```

File: daprdocs/content/en/reference/api/workflow_api.md [hunk 5]
```diff
@@ -0,0 +1,120 @@
...
+```
+### POST terminate workflow request
+```bash
+POST http://localhost:3500/v1.0-alpha1/workflows/{workflowComponent}/{instanceId}/terminate
```
Comment (msfussell) [inline daprdocs/content/en/reference/api/workflow_api.md:38]:
```suggestion
POST http://localhost:3500/v1.0-alpha1/workflows/{name}/{instanceId}/terminate
```

File: daprdocs/content/en/reference/api/workflow_api.md [hunk 6]
```diff
@@ -0,0 +1,120 @@
...
+```
+### GET workflow request
+```bash
+GET http://localhost:3500/v1.0-alpha1/workflows/{workflowComponent}/{workflowType}/{instanceId}
```
Comment (msfussell) [inline daprdocs/content/en/reference/api/workflow_api.md:42]:
```suggestion
GET http://localhost:3500/v1.0-alpha1/workflows/{name}/{workflowType}/{instanceId}
```

**Comment by hhunter-ms** [comment]
@nyemade-uversky @msfussell this PR is purely for the staging URL. If you have any feedback or updates to Workflow API documentation, please do so here: #2915

**Pull request closed without merging**